### ISSUE TITLE
TVOS not matching

### ISSUE THREAD

**Issue by toddpi314** [phonegap/ios-sim#200]
Had to remove the iOS prefix on https://github.com/phonegap/ios-sim/blob/master/src/lib.js#L198

Device: `Apple-TV-1080p`

**Comment by shazron** [comment]
Hi, which command is failing for ios-sim? Trying to repro.

**Comment by shazron** [comment]
Ping.

**Comment by rfsbsb** [comment]
Hi I have tested with the following command `ios-sim launch PROJECT --devicetypeid Apple-TV-1080p` and got the following error: 
***Device id for device name "Apple TV 1080p" and runtime "iOS tvOS 10.1" could not be found, or is not available.***

When I run `ios-sim showdevicetypes` I get:
```
iPhone-5, 10.2
iPhone-5s, 10.2
iPhone-6, 10.2
iPhone-6-Plus, 10.2
iPhone-6s, 10.2
iPhone-6s-Plus, 10.2
iPhone-7, 10.2
iPhone-7-Plus, 10.2
iPhone-SE, 10.2
iPad-Retina, 10.2
iPad-Air, 10.2
iPad-Air-2, 10.2
iPad-Pro--9-7-inch-, 10.2
iPad-Pro, 10.2
Apple-TV-1080p, tvOS 10.1
Apple-Watch-38mm, watchOS 3.1
Apple-Watch-42mm, watchOS 3.1
Apple-Watch-Series-2-38mm, watchOS 3.1
Apple-Watch-Series-2-42mm, watchOS 3.1
```

**Comment by shazron** [comment]
Fixed with #206. Will be in 5.2.0
